### PR TITLE
openHAB docu added

### PR DESCRIPTION
**Pull request by StephanStS**

### CODE REVIEW

File: docs/software/home_automation.md [hunk 1]
```diff
@@ -183,4 +184,42 @@ Also installs:
...
+
+openHAB (open Home Automation Bus) is an open source home automation platform. The *keywords* are *Bindings*, *Things*, *Channels*, *Items*, *Rules*, *Pages*.
+
+![openHAB web interface screenshot](../assets/images/dietpi-software-homeautomation-openhab.jpg){: width="600" height="393" loading="lazy"}
+
+=== "Access to the web interface"
+
+    The web interface is accessible via port **8444**:
+
+    - HTTPS: `https://<your.IP>:8444`
```
Comment (MichaIng) [inline docs/software/home_automation.md:197]:
```suggestion
    - URL: `https://<your.IP>:8444`
```

File: docs/software/home_automation.md [hunk 2]
```diff
@@ -183,4 +184,42 @@ Also installs:
...
+    - Username: `root`
+    - Password = `<your global password>` (default: `dietpi`)
```
Comment (MichaIng) [inline docs/software/home_automation.md:199]: Those are not set on a fresh install but on first web UI access one can choose username and password of the initial admin acccount.
```suggestion
```

File: docs/software/home_automation.md [hunk 3]
```diff
@@ -183,4 +184,42 @@ Also installs:
...
+
+=== "Directories"
+
+    - Install directory: `/usr/share/openhab`
+    - Site configuration directory: `/etc/openhab`
+    - Config file: `/etc/default/openhab`
+    - Backup directory: `/var/lib/openhab/backups`
```
Comment (MichaIng) [inline docs/software/home_automation.md:206]:
```suggestion
    - Data directory: `/var/lib/openhab`
```

File: docs/software/home_automation.md (hunk 2 above)
Comment (StephanStS) [reply docs/software/home_automation.md:199]: Changed with the following push.

**Review by MichaIng — APPROVED** [review]

**Pull request merged**